client/app/components/dynamic-parameters/DynamicParameters.less

```diff
@@ -0,0 +1,25 @@
+.dynamic-value {
+  & ::placeholder, & .dynamic-icon {
+    color: #1890ff !important
+  }
```

Comment:
@arikfr @gabrieldutra Distractingly blue?
<img width="188" alt="Screen Shot 2019-07-18 at 8 19 41" src="https://user-images.githubusercontent.com/486954/61431055-d517a180-a934-11e9-9838-ad3db6b00aab.png">

Some other options:

Option 1
<img width="188" alt="Screen Shot 2019-07-18 at 8 16 39" src="https://user-images.githubusercontent.com/486954/61431028-b9140000-a934-11e9-99ef-1a9b626b75d8.png">

Option 2
<img width="188" alt="Screen Shot 2019-07-18 at 8 17 54" src="https://user-images.githubusercontent.com/486954/61431029-b9140000-a934-11e9-995d-7994d0643af7.png">

Option 3
<img width="188" alt="Screen Shot 2019-07-18 at 8 25 33" src="https://user-images.githubusercontent.com/486954/61431302-ae0d9f80-a935-11e9-893c-08fef5644a4c.png">

I think Option 3. Wdyt?

Reply:
+1 for Option 2.

Reply:
(corrected typo in previous reply)

Reply:
Updated styling to Option 3 for tests. The interesting thing about the prominent blue was that it made me think "there's some 'magic' happening in that parameter".

Reply:
> Updated styling to Option 3 for tests.

👍 

> The interesting thing about the prominent blue was that it made me think "there's some 'magic' happening in that parameter".

I definitely agree but getting the feel for it, I realized it draws too much attention.